client/src/test/scala/eventstore/akka/EventStoreExtensionSpec.scala

```diff
@@ -16,7 +15,7 @@ class EventStoreExtensionSpec extends ActorSpec {
 
     "return connection instance" in new ActorScope {
       val future = EventStoreExtension(system).connection(readEvent)
-      Await.result(future, 3.seconds) must throwAn[EsException]
+      future.await_ must throwAn[EsException]
```

Comment:
Because of specs2 upgrade we somehow need to up the limit of await. Using the one provided via `ActorSpec` that has default await time of six seconds.